src/applications/disability-benefits/all-claims/content/secondaryOtherSources.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import AdditionalInfo from '@department-of-veterans-affairs/formation/AdditionalInfo';
+
+export const otherSourcesDescription = (
+  <div>
+    <h3>Other sources of information</h3>
+    <p>
+      If you were treated at a military or private facility for this event, or
+      reported the event to the authorities or other parties, we can help you
+      gather supporting information from them for your claim.
+    </p>
+    <p>
+      If you have supporting (lay) statements from friends, family, or clergy,
+      or have copies of reports from authorities, you‘ll be able to upload those
```

Comment:
Looks like this apostrophe is backwards. Should be `’`.

Reply:
Dah!! I was on a roll with getting them all lol

Reply:
I tried to whip out a little eslint plugin for this, but couldn't get my AST selectors to work for some reason. I'll write a ticket for this, though, so we can automate this kind of formatting thing.